Unable to build android bindings

I've installed the dependencies and modified the cargo config file as described at: https://github.com/LNP-BP/rgb-node/tree/master/ffi/android/library.

However, I get the following errors when running ```.\gradlew build```.
OS: Windows 10

```
PS E:\android-libs\rgb-node\ffi\android\library> .\gradlew build

> Task :library:compileDebugJavaWithJavac FAILED
E:\android-libs\rgb-node\ffi\android\library\library\src\main\java\org\lnpbp\rgbnode\Runtime.java:9: error: package org.lnpbp.rgbnode_autogen does not exist
import org.lnpbp.rgbnode_autogen.COpaqueStruct;
                                ^
E:\android-libs\rgb-node\ffi\android\library\library\src\main\java\org\lnpbp\rgbnode\Runtime.java:10: error: package org.lnpbp.rgbnode_autogen does not exist
import org.lnpbp.rgbnode_autogen.rgb_node;
                                ^
E:\android-libs\rgb-node\ffi\android\library\library\src\main\java\org\lnpbp\rgbnode\Runtime.java:16: error: cannot find symbol
    private final COpaqueStruct runtime;
                  ^
  symbol:   class COpaqueStruct
  location: class Runtime
E:\android-libs\rgb-node\ffi\android\library\library\src\main\java\org\lnpbp\rgbnode\Runtime.java:25: error: cannot find symbol
            this.runtime = rgb_node.start_rgb(jsonArgs);
                           ^
  symbol:   variable rgb_node
  location: class Runtime
E:\android-libs\rgb-node\ffi\android\library\library\src\main\java\org\lnpbp\rgbnode\Runtime.java:35: error: cannot find symbol
            rgb_node.issue(this.runtime, jsonArgs);
            ^
  symbol:   variable rgb_node
  location: class Runtime
E:\android-libs\rgb-node\ffi\android\library\library\src\main\java\org\lnpbp\rgbnode\Runtime.java:45: error: cannot find symbol
            rgb_node.transfer(this.runtime, jsonArgs);
            ^
  symbol:   variable rgb_node
  location: class Runtime
6 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':library:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 19s
12 actionable tasks: 1 executed, 11 up-to-date
```

Not sure whats going on here to be completely honest. Anyone have any ideas?

Edit: My cargo config file looks like the following (the paths for android targets might seem strange but it's in windows format):
```
[core]
	bare = false
	repositoryformatversion = 0
	filemode = false
	symlinks = false
	ignorecase = true
	logallrefupdates = true

[target.aarch64-linux-android]
ar = "C:\Users\rohan\AppData\Local\Android\Sdk\ndk\21.0.6113669\toolchains\llvm\prebuilt\windows-x86_64\bin\aarch64-linux-android-ar"
linker = "C:\Users\rohan\AppData\Local\Android\Sdk\ndk\21.0.6113669\toolchains\llvm\prebuilt\windows-x86_64\bin\aarch64-linux-android26-clang"

[target.x86_64-linux-android]
ar = "C:\Users\rohan\AppData\Local\Android\Sdk\ndk\21.0.6113669\toolchains\llvm\prebuilt\windows-x86_64\bin\x86_64-linux-android-ar"
linker = "C:\Users\rohan\AppData\Local\Android\Sdk\ndk\21.0.6113669\toolchains\llvm\prebuilt\windows-x86_64\bin\x86_64-linux-android26-clang"

[target.armv7-linux-androideabi]
ar = "C:\Users\rohan\AppData\Local\Android\Sdk\ndk\21.0.6113669\toolchains\llvm\prebuilt\windows-x86_64\bin\arm-linux-androideabi-ar"
linker = "C:\Users\rohan\AppData\Local\Android\Sdk\ndk\21.0.6113669\toolchains\llvm\prebuilt\windows-x86_64\bin\armv7a-linux-androideabi26-clang"

[target.i686-linux-android]
ar = "C:\Users\rohan\AppData\Local\Android\Sdk\ndk\21.0.6113669\toolchains\llvm\prebuilt\windows-x86_64\bin\i686-linux-android-ar"
linker = "C:\Users\rohan\AppData\Local\Android\Sdk\ndk\21.0.6113669\toolchains\llvm\prebuilt\windows-x86_64\bin\i686-linux-android26-clang"
```

--- a/ffi/android/library/library/build.gradle
+++ b/ffi/android/library/library/build.gradle
@@ -17,12 +17,17 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+}
 
-    task buildRust(type: Exec) {
-        workingDir '../'
-        commandLine './build_rust.sh'
-    }
-    build.dependsOn buildRust
+task buildRust(type: Exec) {
+    workingDir '../'
+    commandLine './build_rust.sh'
+}
+
+afterEvaluate {
+  android.libraryVariants.all { variant ->
+    variant.javaCompileProvider.get().dependsOn(buildRust)
+  }
 }
 
 dependencies {

